.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,29 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "17 1 * * *"
+
+permissions:
+  issues: write
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v10
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue was marked stale because it has been labelled with \"question\" for 30 days."
+          close-issue-message: "This issue was closed because it has been 14 days since being marked as stale."
```

Comment:
To be sure, is the issue marked as stale or closed if it is updated? Or does this apply only when there are no updates for longer the specified period?